src/constraints/update/splice/index.js

```diff
@@ -0,0 +1,11 @@
+export default (inverseRelation) => {
+  switch (inverseRelation) {
+    case 'hasMany':
+      return true;
+
+    case 'hasOne':
+    case 'belongsTo':
+    default:
+      return false;
+  }
```

Comment:
Not sure if a `switch` is appropriate for this, considering there are only two outcomes. An `if-else` would suffice. Or even: `return inverseRelation === 'hasMany' ? true : false`.